src/handlebar-helpers.js

```diff
@@ -29,6 +29,14 @@ const helpers = {
       return data.split(delimiter);
     }
     return [data];
+  },
+  divide: function (v1, v2) {
```

Comment:
please use x and y instead of v1 and v2 as this is more common notation for arithmetic than v.
```suggestion
  divide: function (x, y) {
```